components/kubeflow-pipeline/pipeline_components/analytics-server/flwr_analytics_server/fivenum.py

```diff
@@ -0,0 +1,68 @@
+""" 
+
+Five number summary given histogram.   
+Bases for a Boxplot.  
+
+Katulu GmbH  
+(c) 2022  
+
+"""
+
+import numpy as np
+from dataclasses import dataclass
+from typing import List, Optional
+from .fedhist import HistogramData
+
+
+@dataclass
+class fiveNum:
```

Comment:
Nit: Let's use _upper camel case_ here to follow Python convention for classes.

```suggestion
class FiveNum:
```

Reply:
Updated.